client/app/dedicated/server/ftp-backup/access/add/dedicated-server-ftp-backup-access-add.html

```diff
@@ -1,35 +1,46 @@
-<div data-ng-controller="AddAccessFtpBackupCtrl">
-
+<div data-ng-controller="AddAccessFtpBackupCtrl as $ctrl">
     <div data-wizard
-         data-wizard-on-cancel="resetAction"
-         data-wizard-on-finish="addFtpBackup"
+         data-wizard-on-cancel="$ctrl.$scope.resetAction"
+         data-wizard-on-finish="$ctrl.addFtpBackup()"
          data-wizard-title=":: 'server_configuration_ftpbackup_add' | translate"
          data-wizard-confirm-button-text=":: 'common_terminate' | translate"
          data-wizard-hide-cancel-button="true">
 
         <div data-wizard-step
-             data-wizard-step-valid="!loading && access.ip && (access.cifs || access.ftp || access.nfs)"
-             data-wizard-step-on-load="load">
+             data-wizard-step-valid="!$ctrl.loading && $ctrl.access.ip && ($ctrl.access.cifs || $ctrl.access.ftp || $ctrl.access.nfs) && $ctrl.isIpBlocksLengthValid()"
+             data-wizard-step-on-load="$ctrl.load()">
 
             <div class="text-center"
-                 data-ng-if="loading">
+                 data-ng-if="$ctrl.loading">
                 <oui-spinner></oui-spinner>
             </div>
 
-            <div data-ng-if="!loading">
+            <div data-ng-if="!$ctrl.loading">
                 <p data-ng-bind=":: 'server_configuration_ftpbackup_add_info' | translate"></p>
                 <p data-ng-bind=":: 'server_configuration_ftpbackup_add_info3' | translate"></p>
                  <label for="ip"
                         class="control-label"
                         data-translate="server_configuration_ftpbackup_add_ip">
                 </label>
-                <select class="form-control"
-                        id="ip"
-                        name="ip"
-                        multiple
-                        data-ng-model="access.ip"
-                        data-ng-options="ip for ip in access.listIp">
-                </select>
+                <div class="oui-field"
```

Comment:
Wouldn't a `oui-field` work ?

Reply:
Trying to improve `oui-field` to add custom validation on `data-error-messages`

Reply:
@FredericEspiau I've created the jira task UK-88 to handle the improvement, and so, allow this PR to be merged